lib/butler/plugins/cowsay.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Butler.Plugins.Cowsay do
+  use Butler.Plugin
```

Comment:
can we publish the butler plugin api to hex so that external plugins can just require it?